README.md

```diff
@@ -64,13 +64,14 @@ Best is to set up a `.env` file with the settings that need changing (e.g. `POST
 
 ##### PostgreSQL
 
-The PostgreSQL specific settings are the same as for Kartoza's excellent [PostGIS image](https://github.com/kartoza/docker-postgis):
+The PostgreSQL specific settings required to configure your database connection:
```

Comment:
Why remove? I think that's important information for any `docker-compose` setup, as you only have to create one `.env` file.

Reply:
Maybe rephrase yes. Maybe you misunderstood, as it's not very clear: the env var names are the same as in kartoza's image.

Reply:
Yes but no need to point that out? Above it's recommended to use Kartoza, that should be enough.